<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="top_frame">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="logo_label">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>12</pointsize>
              <weight>10</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/logow/Imagens/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_gerar_palpite">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>31</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:hover{	
	color: rgb(0, 0, 0);
	
	background-color: rgb(85, 255, 127);
	
	

	color: #000000f	
	
}

QPushButton{
	
	font: 87 8pt &quot;Arial Black&quot;;	
	
	background-color: rgb(51, 146, 90);

	
	
	color: #fff
	
}
</string>
            </property>
            <property name="text">
             <string>Gerar Palpite</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="down_frame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="entry_1">
            <property name="styleSheet">
             <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
background-color: rgb(51, 146, 90);
color: #ffffff</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>00 - 00 - 00 - 00 - 00 - 00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="entry_2">
            <property name="styleSheet">
             <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
background-color: rgb(51, 146, 90);
color: #ffffff</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>00 - 00 - 00 - 00 - 00 - 00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="entry_3">
            <property name="styleSheet">
             <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
background-color: rgb(51, 146, 90);
color: #ffffff</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>00 - 00 - 00 - 00 - 00 - 00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_info_rodape">
            <property name="styleSheet">
             <string notr="true">font: 87 6pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:7pt; font-weight:600; color:#0000ff;&quot;&gt;Boa Sorte !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="logow.qrc"/>
 </resources>
 <connections/>
</ui>
